src/screens/TopicPageScreen/index.js

```diff
@@ -107,6 +106,12 @@ const TopicPageScreen = (props) => {
     extrapolate: 'clamp'
   });
 
+  const opacityImage = scrollY.interpolate({
+    inputRange: [0, 100],
+    outputRange: [0, 1],
+    extrapolate: 'clamp'
+  });
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `opacityImage` is a good improvement for the user interface. It provides a smooth transition effect when scrolling. However, it's important to ensure that the `scrollY` value is properly initialized and updated during the scroll event to avoid unexpected behavior.

```diff
+  const opacityImage = scrollY.interpolate({
+    inputRange: [0, 100],
+    outputRange: [0, 1],
+    extrapolate: 'clamp'
+  });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->